spec/classes/chrony_spec.rb

```diff
@@ -40,6 +40,22 @@
             it { is_expected.to contain_file('/etc/chrony.keys').with_replace(true) }
             it { is_expected.to contain_file('/etc/chrony.keys').with_content("0 xyzzy\n") }
           end
+        when 'Gentoo'
+          context 'using defaults' do
+            it { is_expected.to contain_file('/etc/chrony/chrony.conf').without_content(%r{^\s*cmdallow}) }
+            ['0.pool.ntp.org', '1.pool.ntp.org', '2.pool.ntp.org', '3.pool.ntp.org'].each do |s|
+              it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*server #{s} iburst$}) }
+            end
```

Comment:
I'd prefer to avoid multiple `it` statements:
```suggestion
            it do
              is_expected.to contain_file('/etc/chrony/chrony.conf')
                .with_content(%r{^\s*server 0.pool.ntp.org iburst$})
                .with_content(%r{^\s*server 1.pool.ntp.org iburst$})
                .with_content(%r{^\s*server 3.pool.ntp.org iburst$})
            end
```

Reply:
Yes, I agree. But then that should be fixed for the other osfamily also. If you like, I can add a commit for that in this PR?